src/components/Navbar.tsx

```
@@ -0,0 +1,47 @@
+'use client'
+import React, { useState } from "react";
+import { HoveredLink, Menu, MenuItem, ProductItem } from "./ui/navbar-menu";
+import { cn } from "@/lib/utils";
+import Link from "next/link";
+
+
+function Navbar({ className }: { className?: string }) {
+    const [active, setActive] = useState<string | null>(null);
+  return (
```

ESLint

[error] Insert `··`

(prettier/prettier)